adobe/spectrum-web-components
# fix(slider): update CSS conversion for more correct visual delivery

## Description
- update space between label and track
- update spacing around `<output>` field
- update spacing after a tick slider that does not have tick labels
- update storybook to better capture some of these changes

## Related issue(s)
- refs #3358

## How has this been tested?
    1. Go [here](https://slider-padding--spectrum-web-components.netlify.app/storybook/?path=/story/slider--default)
    2. See that the labels are closer to the track
    1. Go [here](https://slider-padding--spectrum-web-components.netlify.app/storybook/?path=/story/slider--tick) and [here](https://slider-padding--spectrum-web-components.netlify.app/storybook/?path=/story/slider--tick-labels)
    2. See that the vertical rhythm is different (or the same? specifically the second slider should not be in the same place vertically between the two views) based on the presence of labels

## Types of changes

## Checklist
